docs/versioned_docs/version-2.11.0/deprecated/releases-1x.md

```diff
@@ -0,0 +1,201 @@
+---
+id: releases-1x
+title: OpenEBS 1.x Deprecated Releases
+---
+
+## 1.7.0 - Feb 15 2020
+
+**Change summary:**
+
+- Fixes an issue where Jiva Replicas could get stuck in WO or NA state, when the size of the replica data grows beyond 300GB.
+- Fixes an issue where unused custom resources from older versions are left in the etcd, even after openebs is upgraded. 
+- Fixes an issue where cleanup of Jiva volumes on OpenShift 4.2 environment was failing.
+- Fixes an issue where custom resources used by cStor Volumes fail to get deleted when the underlying pool was removed prior to deleting the volumes.
+- Fixes an issue where a cStor Volume Replica would be incorrectly marked as invalid due to a race condition caused between a terminating and its corresponding newly launched pool pods. 
+
+**Alpha Features**
+
+- Support for generating automated ARM builds for NDM.
+- Support for managing snapshot and clones of ZFS Local PV.
+- Support for setting up PDB and PriorityClasses on cStor Pool Pods. Increasing the e2e coverage and fixing the issue uncovered.
+- Support for resizing Jiva Volume via CSI driver and other bug fixes.
+
+**Additional details:**
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/1.7.0)
+- [Upgrade Steps](/v170/docs/next/upgrade.html)
```

Comment:
it needs to linked to old doc v1.7.0 `https://docs.openebs.io//v170/docs/next/upgrade.html`